content/dev/payment/writing_payment_modules/configuration_keys.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Configuration keys
+description: 
+weight: 5
+---
+
+Payment modules use configuration keys to store the settings for that payment module.
+
+The payment module classes set up some default configuration keys, but it is expected that a module writer will need 
+to set up some of their own keys/settings.
+
+## Default  Keys
+
+The default keys that the system sets up are :-
+
+ - MODULE_PAYMENT_%%_STATUS
+
+   Whether that module is enabled or not
+
+ - MODULE_PAYMENT_%%_SORT_ORDER
+
+   The order that the module appears in both the admin and the catalog
+ - 
+ - MODULE_PAYMENT_%%_ZONE
+
+   Payment modules can be rerstricted to a certain geographic zone
+
+ - MODULE_PAYMENT_%%_DEBUG_MODE
+
+   If debug mode is enabled then log files will be created.
+
+## Custom keys
+
+Most payment modules will need to create their own custom keys/settings. 
+
+to do this the module must add a protected method to its class. 
+
+e.g. 
+
+```angular2html
```

Comment:
```suggestion
```php
```

Reply:
oops. for some reason phpstrm adds the angualar2html by default